src/main/application/schemas/slack_message.sd

```diff
@@ -0,0 +1,86 @@
+schema slack_message {
+
+    field language type string {
+        indexing: "en" | set_language
+    }
+
+    document slack_message {
+
+        field message_id type string {
+            indexing: summary | attribute | index
+        }
+
+        field thread_id type string {
+            indexing: summary | attribute | index
```

Comment:
Add match:word to avoid tokenization, see [match](https://docs.vespa.ai/en/reference/schema-reference.html#match)

Reply:
nice, done